test/full_coverage_path_planner/test_full_coverage_path_planner_plugin.launch

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+
+<launch>
+    <arg name="map" default="$(find full_coverage_path_planner)/maps/grid.yaml"/>
+    <arg name="robot_radius" default="0.3"/>
+    <arg name="tool_radius" default="0.2"/>
+
+    <!--Move base flex, using the full_coverage_path_planner-->
+    <node pkg="mbf_costmap_nav" type="mbf_costmap_nav" respawn="false" name="move_base_flex" output="screen" required="true">
```

Comment:
`[ WARN] [1594969854.211180963]: No plugins loaded at all!`

This file doesn't do anything on my pc

Reply:
This is for the tutorial, you should manually send a goal in rviz to see the global plan. It only configures the global planner plugin. The message "[ WARN] [1594969854.211180963]: No plugins loaded at all!" refers to the local planner.

Reply:
Alright